docs/LowerBound_Dist_Profile_Derivation.ipynb

```diff
@@ -0,0 +1,2319 @@
+{
```

Comment:
<p>Fix spelling of "subsequence"</p><p> </p><p>I would move this comment up by two cells so that it is closer to the <code>\rho_ii</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/586/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
Got it. I will provide the comment right after when `\rho_ii` appears in the calculation.

Reply:
<p>Thanks</p>